modules/tensor_mechanics/src/materials/ComputeMultiPlasticityStress.C

```diff
@@ -673,33 +709,56 @@ ComputeMultiPlasticityStress::canAddConstraints(const std::vector<bool> & act, c
 }
 
 bool
-ComputeMultiPlasticityStress::canChangeScheme(const MooseEnum & current_deactivation_scheme, const bool & can_revert_to_dumb)
+ComputeMultiPlasticityStress::canChangeScheme(DeactivationSchemeEnum current_deactivation_scheme,
+                                              const bool & can_revert_to_dumb)
```

Comment:
No big deal, but a pointer or reference is bigger than a simple type like a bool. Usually you don't need to pass built in types as references. It doesn't save you anything and might actually cost you more if the compiler doesn't optimize that away.